ldms/src/contrib/sampler/variorum_ldms/Makefile.am

```diff
@@ -0,0 +1,16 @@
+pkglib_LTLIBRARIES =
+lib_LTLIBRARIES =
+dist_man7_MANS =
+dist_man1_MANS =
+
+AM_CPPFLAGS = @OVIS_INCLUDE_ABS@
+AM_LDFLAGS = @OVIS_LIB_ABS@
+COMMON_LIBADD = -lsampler_base -lldms -lovis_util -lcoll \
+		@LDFLAGS_GETTIME@
+
+if ENABLE_VARIORUM_LDMS
```

Comment:
Please remove this "if ENABLE_VARIORUM_LDMS" conditional. This check is already being made in the Makefile.am one directory above this. This subdirectory will not be entered if ENABLE_VARIORUM_LDMS is not set.